pkgs/development/python-modules/tank-utility/default.nix

```diff
@@ -0,0 +1,48 @@
+{ lib
+, buildPythonPackage
+, fetchFromGitHub
+, mock
+, pytestCheckHook
+, pythonOlder
+, requests
+, responses
+, urllib3
+}:
+
+buildPythonPackage rec {
+  pname = "tank-utility";
+  version = "1.4.1";
+  format = "setuptools";
+
+  disabled = pythonOlder "3.7";
+
+  src = fetchFromGitHub {
+    owner = "krismolendyke";
+    repo = pname;
+    rev = version;
+    hash = "sha256-2cxAaSyreIzQzCUtiolEV7JbGFKL8Mob3337J0jlMsU=";
+  };
+
+  propagatedBuildInputs = [
+    requests
+    urllib3.optional-dependencies.secure
```

Comment:
This creates a nested list, concat the lists instead.

Reply:
Thanks